.travis.yml

```diff
@@ -56,6 +56,8 @@ matrix:
     - python: 2.7
       env:
         - INSTALL_TYPE=wheel
+        # Dependency checking should get all needed dependencies
+        - DEPENDS=
```

Comment:
Is this supposed to undo the setting on L14? I am not sure that actually takes on Travis. It seems to still get the global setting somehow: https://travis-ci.org/nipy/dipy/jobs/118363119#L132, and I couldn't find any evidence that it gets unset.